### PR TITLE
Epics v7

### PR DESCRIPTION
**Pull request by SkyToGround**
* Updated to EPICS 7 plus related changes.
* Some code was deleted, please check that functionality has not been lost.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,31 @@ It requires the following items to be installed:
...
+1. Edit line 6 of the *conanfile.py*-file to the version of EPICS base that you want to package.
+
+2. Edit line 14 of the *conanfile.py*-file to set the version of the new conan package.
+
+3. Edit line 40 of the *conanfile.py*-file to set the hash of the compressed file. The hash can be determined from running the command `shasum -a 256 base-x.y.z.tar.gz`.
+
+4. When in the directory of the local copy of *conan-epics*, execute this command:
+
+	```
+	conan create . epics/x.y.z-dm1@ess-dmsc/stable
```
Comment (amues) [inline README.md]: Are the package name and version included in this command for a specific reason? You can run `conan create . ess-dmsc/stable` and Conan will pick up the name and version from the conanfile.

**Review by amues — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (SkyToGround) [reply README.md]: Ok, I did not know that. I will change it.

Comment (SkyToGround) [inline README.md]:
```suggestion
	conan create . ess-dmsc/stable
```

File: README.md [hunk 2]
```diff
@@ -18,6 +18,31 @@ It requires the following items to be installed:
...
+2. Edit line 14 of the *conanfile.py*-file to set the version of the new conan package.
+
+3. Edit line 40 of the *conanfile.py*-file to set the hash of the compressed file. The hash can be determined from running the command `shasum -a 256 base-x.y.z.tar.gz`.
+
+4. When in the directory of the local copy of *conan-epics*, execute this command:
+
+	```
+	conan create . ess-dmsc/stable
+	```
+	Where **x.y.z-dm1** is the same version string as set on line 14 in the *conanfile.py*-file.
```
Comment (SkyToGround) [inline README.md]:
```suggestion
```

**Comment by amues** [comment]
By the way, I had to upgrade my perl installation for the package to build on macOS. I don't know how relevant this information is for the README.

**Comment by SkyToGround** [comment]
@amues That might be significant, I will ask you about the details around that next week. For now, have a nice weekend.

**Pull request merged**